securedrop/tests/functional/source_navigation_steps.py

```diff
@@ -161,5 +161,4 @@ def _source_deletes_a_journalist_reply(self):
 
     def _source_logs_out(self):
         self.driver.find_element_by_id('logout').click()
-        notification = self.driver.find_element_by_css_selector('.important')
-        assert 'Thank you for exiting your session!' in notification.text
+        self.driver.find_element_by_css_selector('.important')
```

Comment:
This is an effective test (`find_element_by_css_selector` will raise `NoSuchElementException` if it cannot find any matching elements, which causes a test failure), but FWIW I generally prefer test assertions to be more explicit. I think that's a little harder to do in this case because this class doesn't inherit from `unittest.TestCase` or similar, and therefore doesn't provide as many useful assertion test methods.